src/config.ts

```diff
@@ -152,6 +155,36 @@ function loadBootstrapOptions(
     return undefined;
 }
 
+function loadCorsOptions(option: IProxyOption): CorsOptions {
+    if (option.cors) {
+        return option.cors;
+    }
+
+    const computedCorsOptions: CorsOptions = {
+        origin: process.env.CORS_ORIGIN || '*',
+        maxAge: safeNumber(process.env.CORS_MAX_AGE, 172800),
+        exposedHeaders: 'ETag',
+    };
+
+    // if cors origin provided contains "," it means it's a list of urls, transform to array
```

Comment:
can't we always produce an array?

Reply:
the behaviour of the cors module is different when providing an array vs a string, for example by setting the origin as `"https://example.com"` (string) you are allowed to make any requests from `"https://example.com"` and beyond (e.g. `"https://example.com/*"`), instead by using an array you need to specify the exact URLs that are allowed in CORS, for example: `["https://example.com/mypage"]` allows only requests with the same `https://example.com/mypage` origin, you can learn more here https://github.com/expressjs/cors/issues/185